halo/attest/keeper/common_test.go

```diff
@@ -0,0 +1,62 @@
+package keeper_test
+
+import (
+	"testing"
+
+	"github.com/omni-network/omni/halo/attest/keeper"
+	"github.com/omni-network/omni/halo/attest/testutil"
+	"github.com/omni-network/omni/halo/attest/types"
+
+	"cosmossdk.io/core/store"
+	storetypes "cosmossdk.io/store/types"
+	"github.com/cosmos/cosmos-sdk/runtime"
+	ctestutil "github.com/cosmos/cosmos-sdk/testutil"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/mock/gomock"
+)
+
+type mocks struct {
```

Comment:
what is the point of this struct? Doesn't seem like it is used for anything really?\